schwifty/bank_registry/manual_it.json

```diff
@@ -62,5 +62,229 @@
     "name": "POSTE ITALIANE SPA",
     "short_name": "POSTE ITALIANE SPA",
     "primary": true
+  },
+  {
+    "country_code": "IT",
+    "bic": "CRGEITGG",
+    "bank_code": "CRGE",
+    "name": "Banca Carige",
+    "short_name": "Banca Carige",
+    "primary": true
+  },
+  {
+    "country_code": "IT",
+    "bic": "CCRTIT2T",
+    "bank_code": "CCRT",
+    "name": "Banca di Credito Cooperativo",
+    "short_name": "Banca di Credito Cooperativo",
+    "primary": true
+  },
+  {
+    "country_code": "IT",
+    "bic": "BCITITMM",
+    "bank_code": "BCIT",
```

Comment:
Also for Italian banks, the domestic bank code is in fact numeric. It has to match the bank code part of the IBAN or otherwise schwift wouldn't be able to find the BIC related to a given IBAN.

Reply:
@mdomke feel free to take another look. I have added all the correct bank codes.
Also took the liberty to sort all banks with bank code asc as well as inner json attributes for uniformity and better readability
let me know what you think